gtests/net/mptcp/add_addr/add_addr_server.pkt

```diff
@@ -16,9 +16,14 @@
 +0.01   < . 1:1(0) ack 1 win 257 <mpcapable v1 flags[flag_h] key[ckey=2,skey]>
 +0    accept(3, ..., ...) = 4
 
+// ADD_ADDR is sent directly (>= 5.12), not after first data (< 5.12)
++0     > . 1:1(0) ack 1 <add_address addr[saddr] hmac=auto,dss dack4=1 ssn=1 dll=0 nocs>
+// TODO: support injecting ADD_ADDR echo
```

Comment:
Linked to https://github.com/multipath-tcp/mptcp_net-next/issues/142
(same for the others)